src/prefect/utilities/logging.py

```diff
@@ -139,7 +139,11 @@ class CloudHandler(logging.Handler):
     def emit(self, record: logging.LogRecord) -> None:  # type: ignore
         """Emit a new log"""
         # if we shouldn't log to cloud, don't emit
-        if not context.config.logging.log_to_cloud:
+        if not context.config.cloud.send_flow_run_logs:
+            return
+
+        # if its not during a backend flow run, don't emit
+        if not context.get("running_with_backend"):
```

Comment:
Because you are also relying on this context variable, you are going to miss at least two logs that I can see, and any errors that occur prior to this being set

Reply:
Could you point out some of the logs you're talking about so I can assess the best place to address them? I think the one line change at https://github.com/PrefectHQ/prefect/blob/cd115113b6962a7107b92fcd692b2131191feba1/src/prefect/cli/execute.py#L56-L57 should cover most everything.

Reply:
Yea sorry about that:
- https://github.com/PrefectHQ/prefect/blob/master/src/prefect/engine/flow_runner.py#L245
- https://github.com/PrefectHQ/prefect/blob/master/src/prefect/engine/flow_runner.py#L261

I didn't realize this value was set so early on (I thought it was only applied once the flow runner began), so maybe these logs are safe